bioimageio/spec/model/v0_4/schema.py

```diff
@@ -424,24 +437,53 @@ def get_tnames(tname: str):
             return [t.get("name") if isinstance(t, dict) else t.name for t in data.get(tname, [])]
 
         valid_input_tensor_references = get_tnames("inputs")
-        # a model is not allowed to have no or empty outputs, but I hope that this is validated somewhere else,
-        # so I use get with a default empty list here
-        for out in data.get("outputs", []):
-            if out.postprocessing is missing_:
+        ins = data.get("inputs", [])
+        outs = data.get("outputs", [])
+        if not isinstance(ins, list) or not isinstance(outs, list):
+            raise ValidationError(
```

Comment:
Should this ever happen? I.e. shouldn't the other ValidationErrors be encountered earlier?
If that's the case, then we should maybe throw some other error here, prob. `RuntimeError`.

Reply:
the code is executed anyway, because we generate the dict structure with all errors and don't just raise on the first...

Reply:
ok, then it should be fine..